source/php-libraries.txt

```diff
@@ -12,133 +12,83 @@ PHP Libraries, Frameworks, and Tools
+  - `Laminas <https://github.com/doctrine/DoctrineMongoODMModule>`__ (formerly Zend Framework)
 
-- `Mongo Queue PHP <https://github.com/traderinteractive/mongo-queue-php>`_ is
-  a PHP message queue, which uses MongoDB as a backend.
+- `Mongo PHP Adapter <https://github.com/alcaeus/mongo-php-adapter>`__ is a
+  userland library designed to act as an adapter between applications that rely
+  on the legacy ``mongo`` extension and the new ``mongodb`` extension. It
+  offers the API of the legacy driver for the new driver and library.
 
-- `Mongo PHP Adapter <https://github.com/alcaeus/mongo-php-adapter>`_ is a
-  userland library designed to act as an adapter between applications relying on
-  the legacy ``mongo`` extension and the new ``mongodb`` extension. It provides
-  the API of the legacy driver on top of the new driver and library, which
-  allows for compatibility with PHP 7.
+- `Mongolid <https://github.com/leroy-merlin-br/mongolid>`__ is a performant
+  ODM for PHP and MongoDB. It implements both ActiveRecord and DataMapper
+  design patterns and supports embedded and referenced documents. You can use
+  this library standalone or with the `Laravel <https://github.com/leroy-merlin-br/mongolid-laravel>`__
+  integration.
 
-- `Mongolid <https://github.com/leroy-merlin-br/mongolid>`_: A fast ODM for PHP and
-  MongoDB, which implements both ActiveRecord and DataMapper design patterns and
-  supports embedded and referenced documents. An integration with
-  `Laravel <https://github.com/leroy-merlin-br/mongolid-laravel>`__ is also available.
-
-- `Yadm <https://github.com/makasim/yadm>`_ is a MongoDB ODM written for the
-  **mongodb** extension. It is schema-less and supports fast object hydration
-  and persistence, which makes it well-suited for modeling aggregation results.
-
-- `Xenus <https://github.com/abellion/xenus>`_ is an elegant MongoDB ODM
-  that supports events, relationships, embedded documents, and more. An
-  integration with `Laravel <https://github.com/abellion/xenus-laravel>`__ is
-  also available, which adds support for failed jobs, migrations, and events.
+- `Xenus <https://github.com/abellion/xenus>`__ is a MongoDB ODM
+  that supports events, relationships, embedded documents, and more. You can
+  use this library standalone or with the `Laravel <https://github.com/abellion/xenus-laravel>`__
+  integration which adds support for failed jobs, migrations, and events.
 
 Framework Integrations
-~~~~~~~~~~~~~~~~~~~~~~
+----------------------
 
 - Drupal
 
-  - `MongoDB integration for Drupal <https://www.drupal.org/project/mongodb>`_.
+  - `MongoDB integration for Drupal <https://www.drupal.org/project/mongodb>`__.
     This is a collection of several modules which allow sites to store different
-    types of Drupal data in MongoDB. Support for the ``mongodb`` extension
-    exists for Drupal 8+.
+    types of Drupal data in MongoDB. The ``mongodb`` extension supports
```

Comment:
Out of scope but on the topic of restrictive clauses, this sentence should read, "This is a collection of several modules _that_ allow..."